### ISSUE TITLE
The file of model

### ISSUE THREAD

**Issue by a962097364** [zgojcic/Rigid3DSceneFlow#9]
Hello! I want to change the model_best.pt file into .onnx file, however, I don't clearly find out the input parameter of network.
It will help a lot if you can answer me the input parameter of the network.
If you can offer the .onnx file of the model will help me a lot.

Thank you!

**Comment by zgojcic** [comment]
Hi, I am not sure if I fully understand your question, but the `model_best.pt` file also includes other information e.g. the optimizer and the scheduler parameters.

To extract the model parameters, you can run `model['state_dict']` which will return a dictionary of all parameters. If you are then interested in the weights of the first conv layer in the encoder, you can obtain them as 'model['state_dict']['encoder.conv1.kernel']' which is a tensor of weights with a shape 'model['state_dict']['encoder.conv1.kernel'].shape'

Hope that this helps. 

Zan

**Comment by a962097364** [comment]
It really helps me, thanks a lot!
